crypto/pem/pem_lib.c

```diff
@@ -331,7 +331,7 @@ int PEM_ASN1_write_bio(i2d_of_void *i2d, const char *name, BIO *bp,
 
     if (enc != NULL) {
         objstr = EVP_CIPHER_get0_name(enc);
-        if (objstr == NULL || EVP_CIPHER_get_iv_length(enc) == 0
+        if (objstr == NULL || EVP_CIPHER_get_iv_length(enc) <= 0
```

Comment:
I am not sure there should even be a lower bound check here anymore.

Reply:
This is no longer present, it was removed in the latest fixup

Reply:
removed